pkg/check/act/act.go

```diff
@@ -0,0 +1,179 @@
+package act
+
+import (
+	"bytes"
+	"context"
+	"errors"
+	"fmt"
+	"time"
+
+	"github.com/ethersphere/bee/pkg/swarm"
+	"github.com/ethersphere/beekeeper/pkg/bee"
+	"github.com/ethersphere/beekeeper/pkg/bee/api"
+	"github.com/ethersphere/beekeeper/pkg/beekeeper"
+	"github.com/ethersphere/beekeeper/pkg/logging"
+	"github.com/ethersphere/beekeeper/pkg/orchestration"
+	"github.com/ethersphere/beekeeper/pkg/random"
+)
+
+// Options represents check options
+type Options struct {
+	FileName      string
+	FileSize      int64
+	PostageAmount int64
+	PostageDepth  uint64
+	PostageLabel  string
+	Seed          int64
+}
+
+// NewDefaultOptions returns new default options
+func NewDefaultOptions() Options {
+	return Options{
+		FileName:      "act",
+		FileSize:      1 * 1024,
+		PostageAmount: 420000000,
+		PostageDepth:  17,
+		PostageLabel:  "act-label",
+		Seed:          0,
+	}
+}
+
+// compile check whether Check implements interface
+var _ beekeeper.Action = (*Check)(nil)
+
+// Check instance
+type Check struct {
+	logger logging.Logger
+}
+
+// NewCheck returns new check
+func NewCheck(logger logging.Logger) beekeeper.Action {
+	return &Check{
+		logger: logger,
+	}
+}
+
+// Run executes act check
+func (c *Check) Run(ctx context.Context, cluster orchestration.Cluster, opts interface{}) (err error) {
+	c.logger.Info("ACT test started")
```

Comment:
this log is redundant and can be removed, every check it is already printed here https://github.com/ethersphere/beekeeper/blob/b017aef7386afc79d566df09085645f4d492a6b2/cmd/beekeeper/cmd/check.go#L123

Reply:
removed

https://github.com/Solar-Punk-Ltd/beekeeper/blob/feat/act/pkg/check/act/act.go#L58